### ISSUE TITLE
Pellino family (IPR006800)

### ISSUE THREAD

**Issue by hattrill** [geneontology/go-annotation#2218]
Pellino family (IPR006800) is associated with "Toll signaling pathway" GO:0008063, but this family of proteins has been shown to both positively and negatively regulate Toll signaling. 

The Pellino family do have ubiquitin ligase activity (PMID:26085209,  PMID:1767529), so possibilty to link this domain with terms for this activity and process.

**Comment by asangrador** [comment]
Yes, I have updated the annotation for this entry and GO terms, adding 'ubiquitin protein ligase activity' and 'protein polyubiquitination' and changing the Toll signalling term to 'regulation of Toll signaling pathway'.
Thanks,
